client/view.js

```diff
@@ -3,29 +3,35 @@ export default class View
 	#bus = null;
 	#playerRows = {};
 	#playerId = null;
+	#playerName = null;
 
 	constructor($container, bus, playerId)
 	{
 		this.#bus = bus;
 		this.#initBus();
-
 		this.#playerId = playerId;
 
 		$container.querySelector('button.js-create-game').addEventListener('click', e => {
-			this.#bus.publish('create-game');
+			e.preventDefault();
+			this.#bus.publish('create-game',this.playerName.value);
 		}, false);
 
 		$container.querySelector('button.js-join-game').addEventListener('click', e => {
 			e.preventDefault();
-			this.#bus.publish('join-game', $container.querySelector('input[name=game-id]').value);
+			this.#bus.publish('join-game', 
+			$container.querySelector('input[name=game-id]').value,
+			this.playerName.value
+			);
 		}, false);
 
+		this.playerName = $container.querySelector('#txtLobbyPlayerName')

```

Comment:
the current pattern is to use a `$` prefix to denote variables which represent DOM elements. we can change this to an `El` suffix (or other) if we like, but we need to be consistent. Rename to `$playerName` (and update all usages)